modules/tribe/module.info

```diff
@@ -0,0 +1,6 @@
+{
+    "id" : "tribe",
+    "version" : "0.0.1",
+    "displayName" : "tribe Faction",

```

Comment:
The T should be capitalized, since `displayName` is the string displayed to the user.